examples/offline_inference.py

```diff
@@ -1,22 +1,80 @@
+from dataclasses import asdict
+
 from vllm import LLM, SamplingParams
+from vllm.engine.arg_utils import EngineArgs
+from vllm.utils import FlexibleArgumentParser
+
+
+def get_prompts(args):
+    # The default sample prompts.
+    prompts = [
+        "Hello, my name is",
+        "The president of the United States is",
+        "The capital of France is",
+        "The future of AI is",
+    ]
+
+    if args.num_prompts != len(prompts):
+        prompts = (prompts *
+                   ((args.num_prompts // len(prompts)) + 1))[:args.num_prompts]
+
+    return prompts
```

Comment:
```suggestion
def get_prompts(num_prompts: int):
    # The default sample prompts.
    prompts = [
        "Hello, my name is",
        "The president of the United States is",
        "The capital of France is",
        "The future of AI is",
    ]

    if num_prompts != len(prompts):
        prompts = (prompts * (num_prompts // len(prompts) + 1))[:num_prompts]

    return prompts
```
There is no need to parse whole args to this function.